Microsoft.Toolkit.Uwp.Services/Services/Bing/BingDataProvider.cs

```diff
@@ -42,9 +42,10 @@ public class BingDataProvider : DataProviderBase<BingSearchConfig, BingResult>
         {
             var countryValue = config.Country.GetStringValue();
             var locParameter = string.IsNullOrEmpty(countryValue) ? string.Empty : $"loc:{countryValue}+";
+            var newsParameter = !config.News ? string.Empty : "news/";
```

Comment:
Can this conditional be swapped round please so that it's:
config.News ? "news/" : string.Empty;